app/views/todo_mailer/weekly_report.html.erb

```diff
@@ -1,12 +1,22 @@
-<h1>Hi, <%= @user.email %></h1>
+<h1>Hi, <%= @user.nombre %></h1>
```

Comment:
to have a more consistent code, English should be used to name all variables, functions and params.

Reply:
yes, i got it